client/src/common_css/flexbox-grid.scss

```diff
@@ -1,988 +0,0 @@
-    -ms-flex-preferred-size: 75%;
-    flex-basis: 75%;
-    max-width: 75%;
-  }
-
-  .fcol-md-10 {
-    -ms-flex-preferred-size: 83.33333333%;
-    flex-basis: 83.33333333%;
-    max-width: 83.33333333%;
-  }
-
-  .fcol-md-11 {
-    -ms-flex-preferred-size: 91.66666667%;
-    flex-basis: 91.66666667%;
-    max-width: 91.66666667%;
-  }
-
-  .fcol-md-12 {
-    -ms-flex-preferred-size: 100%;
-    flex-basis: 100%;
-    max-width: 100%;
-  }
-
-  .fcol-md-offset-0 {
-    margin-left: 0;
-  }
-
-  .fcol-md-offset-1 {
-    margin-left: 8.33333333%;
-  }
-
-  .fcol-md-offset-2 {
-    margin-left: 16.66666667%;
-  }
-
-  .fcol-md-offset-3 {
-    margin-left: 25%;
-  }
-
-  .fcol-md-offset-4 {
-    margin-left: 33.33333333%;
-  }
-
-  .fcol-md-offset-5 {
-    margin-left: 41.66666667%;
-  }
-
-  .fcol-md-offset-6 {
-    margin-left: 50%;
-  }
-
-  .fcol-md-offset-7 {
-    margin-left: 58.33333333%;
-  }
-
-  .fcol-md-offset-8 {
-    margin-left: 66.66666667%;
-  }
-
-  .fcol-md-offset-9 {
-    margin-left: 75%;
-  }
-
-  .fcol-md-offset-10 {
-    margin-left: 83.33333333%;
-  }
-
-  .fcol-md-offset-11 {
-    margin-left: 91.66666667%;
-  }
-
-  .start-md {
-    -webkit-box-pack: start;
-    -ms-flex-pack: start;
-    justify-content: flex-start;
-    text-align: start;
-  }
-
-  .center-md {
-    -webkit-box-pack: center;
-    -ms-flex-pack: center;
-    justify-content: center;
-    text-align: center;
-  }
-
-  .end-md {
-    -webkit-box-pack: end;
-    -ms-flex-pack: end;
-    justify-content: flex-end;
-    text-align: end;
-  }
-
-  .top-md {
-    -webkit-box-align: start;
-    -ms-flex-align: start;
-    align-items: flex-start;
-  }
-
-  .middle-md {
-    -webkit-box-align: center;
-    -ms-flex-align: center;
-    align-items: center;
-  }
-
-  .bottom-md {
-    -webkit-box-align: end;
-    -ms-flex-align: end;
-    align-items: flex-end;
-  }
-
-  .around-md {
-    -ms-flex-pack: distribute;
-    justify-content: space-around;
-  }
-
-  .between-md {
-    -webkit-box-pack: justify;
-    -ms-flex-pack: justify;
-    justify-content: space-between;
-  }
-
-  .first-md {
-    -webkit-box-ordinal-group: 0;
-    -ms-flex-order: -1;
-    order: -1;
-  }
-
-  .last-md {
-    -webkit-box-ordinal-group: 2;
-    -ms-flex-order: 1;
-    order: 1;
-  }
-}
-
-@media only screen and (min-width: $minLargeDevice) {
-
-  .fcol-lg,
-  .fcol-lg-1,
-  .fcol-lg-2,
-  .fcol-lg-3,
-  .fcol-lg-4,
-  .fcol-lg-5,
-  .fcol-lg-6,
-  .fcol-lg-7,
-  .fcol-lg-8,
-  .fcol-lg-9,
-  .fcol-lg-10,
-  .fcol-lg-11,
-  .fcol-lg-12,
-  .fcol-lg-offset-0,
-  .fcol-lg-offset-1,
-  .fcol-lg-offset-2,
-  .fcol-lg-offset-3,
-  .fcol-lg-offset-4,
-  .fcol-lg-offset-5,
-  .fcol-lg-offset-6,
-  .fcol-lg-offset-7,
-  .fcol-lg-offset-8,
-  .fcol-lg-offset-9,
-  .fcol-lg-offset-10,
-  .fcol-lg-offset-11,
-  .fcol-lg-offset-12 {
-    box-sizing: border-box;
-    -webkit-box-flex: 0;
-    -ms-flex: 0 0 auto;
-    flex: 0 0 auto;
-    padding-right: 0.5rem;
-    padding-left: 0.5rem;
```

Comment:
I think we lost this general padding, might be places where we counted on it in our designs.
![image](https://user-images.githubusercontent.com/11301919/115437966-d85f5180-a1da-11eb-8fa1-961758593c26.png)

Probably want to bring it back for bootstrap cols in `bootstrap-wet-fixes-extensions.scss`

Reply:
`.row` has a negative margin, so I just removed those margins instead. Should have about the same impact I think?

Reply:
I took a closer look, I guess our `frow`s were using -0.5rem margins for some reason (computed value of -5px). Bootstrap `container`s have 15px (and their `row`s have -15px), so our `frow`s all effectively had a 10px margin.

Looks like the -15px margins are important to bootstraps grid layout concept though ([link](https://stackoverflow.com/questions/51282923/bootstrap-negative-margin-on-rows)). Since we're committing to Bootstrap, we shouldn't be messing with that. Proper solution would be to manually review and add back in util classes to explicitly add the desired padding on rows (like the above home page example) where the padding is important for the site to look good. I'd guess that most grid system content doesn't actually look all that different with the implicit padding of the `frow`s effectively removed.